packages/web/src/components/notification/Notifications/UserProfilePictureList.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+import { User } from 'common/models/User'
+import { formatCount } from 'common/utils/formatUtil'
+import Tooltip from 'components/tooltip/Tooltip'
+
+import { ProfilePicture } from './ProfilePicture'
+import styles from './UserProfilePictureList.module.css'
+
+const messages = {
+  viewAllTooltip: 'View All'
+}
+
+const USER_LENGTH_LIMIT = 8
+
+type UserProfileListProps = {
+  users: Array<User>
+}
+
+export const UserProfilePictureList = ({ users }: UserProfileListProps) => {
+  const showUserListModal = users.length > USER_LENGTH_LIMIT
```

Comment:
we could make USER_LENGTH_LIMIT a prop so that diff places (e.g. top followers in tipping) using this list can control what the cutoff is